### DISCUSSION TITLE
Error when recompiling documentations

### DISCUSSION THREAD

**Discussion by EmmanuelHape** [IBAHCM/RPiR#229] [Archive] [answered]
@richardreeve @soniamitchell 

Hi Sonia, I have managed to reset everything and now my R is running well. 

I am running devtools::document() to recompile my documentations and I am getting this error, tried to check it but I didn't manage to solve it.

Error in read.dcf(path_desc) : 
  Line starting 'creating model funct ...' is malformed!

Help !!

**Comment by EmmanuelHape** [comment, marked as answer]
Solved it, thanks!

**Reply by soniamitchell** [reply]
Sorry we didn't get to this earlier. Glad you've got it fixed though! 👍
